sa-gateway/src/main/resources/application-test.yaml

```diff
@@ -0,0 +1,7 @@
+spring:
+  zipkin:
+    enabled: false
+eureka:

```

Comment:
As I mentined before, you could remove Eureka at all, because you rely on K8S service discovery now.